test/connector/exchange/loopring/test_loopring_market.py

```diff
@@ -22,7 +22,7 @@
     OrderCancelledEvent,
     TradeType,
 )
-from hummingbot.core.data_type.trade_fee import TradeFee
+from hummingbot.core.data_type.trade_fee import AddedToCostTradeFee
```

Comment:
`hummingbot.core` should come after

Reply:
As discussed elsewhere, will not sort here as these tests are not maintained